en/docs/install-and-setup/setup/api-controller/encrypting-secrets-with-ctl.md

```diff
@@ -0,0 +1,125 @@
+# Encrypting Secrets with CTL
```

Comment:
@SanojPunchihewa / @uvindra / @samgnaniah 
This is only applicable to the MI component yeah?
I think we will have to group the MI-specific content separately here?
Shall we use "Integration Controller" or "Managing Integrations" as the umbrella topic?

Reply:
Ok. I think we already have "managing integrations with the CLI" as a topic.

Reply:
We can use this in APIM as well, therefore @uvindra suggested to have it under a separate section.

Reply:
No this actually generic, so this can be applied to both APIM and MI. Hence we thought of having it this way so we can leverage it for APIM as well